src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.Android.cs

```diff
@@ -15,8 +15,14 @@ public partial class FlipView : Selector
 	{
 		private NativePagedView PagedView { get { return InternalItemsPanelRoot as NativePagedView; } }
 
+		partial void InitializePartial()
+		{
+			SetClipChildren(true);
```

Comment:
@ajpinedam looks good. Is there a way to validate this in a runtime test?

Reply:
Added a runtime test to validate the flag is being set, but also modified a manual test to include what we expect.